### DISCUSSION TITLE
Method for DNA extraction of hemolymph

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1418] [Q&A]
What would be the recommended means to extract DNA from archived crab hemolymph samples in EtOH?

**Comment by kubu4** [comment]
Pellet, remove supernatant, extract DNA using preferred method (DNAzol, Qiagen DNeasy, Zymo Quick DNA Miniprep kit).

Don't really have a recommendation here, although using one of the kits is a little more "foolproof", as there's no precipitation/pellet washing/etc).
